### PR TITLE
update fury rotation

### PR DESCRIPTION
**Pull request by codengine**

### CODE REVIEW

File: Specialization/Fury.lua [hunk 1]
```diff
@@ -1,164 +1,223 @@
...
     end
 
-    return FR.Slam;
+    -- rampage
+    if talents[FR.Rampage] and rage >= 80 then
+        return FR.Rampage
+    end
+
+    -- slam,if=talent.annihilator
+    if rage >= 20 and (talents[FR.Annihilator]) then
```
Comment (kojakk) [inline Specialization/Fury.lua:202]: I find casting slam in multi target situations to be a bit suspect. (even with annihilator). I'm going to look into it a bit.

File: Specialization/Fury.lua [hunk 2]
```diff
@@ -1,164 +1,223 @@
...
+    if not recklessnessEmpowered and talents[FR.RagingBlow] and cooldown[FR.RagingBlow].ready and (cooldown[FR.RagingBlow].charges > 1) then
+        return FR.RagingBlow
     end
-	
-	if not talents[FR.TitanicRage] and cooldown[FR.RagingBlowEmpowered].charges >= 1 then
-        return FR.RagingBlowEmpowered;
+
+    -- crushing_blow,if=charges>1
+    if recklessnessEmpowered and talents[FR.RagingBlow] and cooldown[FR.CrushingBlow].ready and (cooldown[FR.CrushingBlow].charges > 1) then
+        return FR.CrushingBlow
```
Comment (kojakk) [inline Specialization/Fury.lua:181]: Your crushing blow won't cast correctly for the Odyns Fury version.

**Review by kojakk — CHANGES_REQUESTED** [review]

File: Specialization/Fury.lua (hunk 1 above)
Comment (codengine) [reply Specialization/Fury.lua:202]: With "Improved Whirlwind" every single target spell also affects 2-4 other targets. So this alone is not a counter argument against using slam ;)

Comment (kojakk) [reply Specialization/Fury.lua:202]: ahh, yes. Thank you. That clears up my confusion.

**Review by kojakk — APPROVED** [review]
The beta removes crushing blow from the Titanic Fury talent

**Pull request merged**